dev/bots/suite_runners/run_web_tests.dart

```diff
@@ -197,48 +197,13 @@ class WebTestsSuite {
       () => _runGalleryE2eWebTest('debug'),
       () => _runGalleryE2eWebTest('profile'),
       () => _runGalleryE2eWebTest('release'),
-      () =>
-          runWebServiceWorkerTest(headless: true, testType: ServiceWorkerTestType.withoutFlutterJs),
-      () => runWebServiceWorkerTest(headless: true, testType: ServiceWorkerTestType.withFlutterJs),
-      () => runWebServiceWorkerTest(
-        headless: true,
-        testType: ServiceWorkerTestType.withFlutterJsShort,
-      ),
-      () => runWebServiceWorkerTest(
-        headless: true,
-        testType: ServiceWorkerTestType.withFlutterJsEntrypointLoadedEvent,
-      ),
-      () => runWebServiceWorkerTest(
-        headless: true,
-        testType: ServiceWorkerTestType.withFlutterJsTrustedTypesOn,
-      ),
-      () => runWebServiceWorkerTest(
-        headless: true,
-        testType: ServiceWorkerTestType.withFlutterJsNonceOn,
-      ),
-      () => runWebServiceWorkerTestWithCachingResources(
-        headless: true,
-        testType: ServiceWorkerTestType.withoutFlutterJs,
-      ),
-      () => runWebServiceWorkerTestWithCachingResources(
-        headless: true,
-        testType: ServiceWorkerTestType.withFlutterJs,
-      ),
-      () => runWebServiceWorkerTestWithCachingResources(
-        headless: true,
-        testType: ServiceWorkerTestType.withFlutterJsShort,
-      ),
-      () => runWebServiceWorkerTestWithCachingResources(
-        headless: true,
-        testType: ServiceWorkerTestType.withFlutterJsEntrypointLoadedEvent,
-      ),
-      () => runWebServiceWorkerTestWithCachingResources(
-        headless: true,
-        testType: ServiceWorkerTestType.withFlutterJsTrustedTypesOn,
-      ),
-      () => runWebServiceWorkerTestWithGeneratedEntrypoint(headless: true),
-      () => runWebServiceWorkerTestWithBlockedServiceWorkers(headless: true),
-      () => runWebServiceWorkerTestWithCustomServiceWorkerVersion(headless: true),
+      () => runServiceWorkerCleanupTest(headless: true),
+      () => runWebCanvasKitUnitTests(),
+      () => runWebSkwasmUnitTests(),
```

Comment:
These 2 aren't supposed to be run as part of this shard. They are run separately as you can see here:
https://github.com/flutter/flutter/blob/c932faf754723287ebf8fc64bbb72a85046f6afb/dev/bots/test.dart#L134-L139